spec/lib/mtg_json_importer_spec.rb

```diff
@@ -0,0 +1,150 @@
+require 'rails_helper'
+require 'mtg_json_importer'
+
+RSpec.describe MtgJsonImporter do
+  describe '.run' do
+    context 'given json for a set' do
```

Comment:
I'm not sure we need contexts for this spec, I think it should be one scenario using the JSON data _with_ the card data. Maybe use describe blocks to neatly divide the set and card based specs. WDYT?